src/views/account/Attendance/Atoms/AttendanceItem.tsx

```diff
@@ -74,14 +74,16 @@ const AttendanceItem: React.FC<AttendanceItemProps> = ({
         let totalTime = "";
         if ("hours" in item) {
           const [hours, minutes] = item.hours.split("h").map(Number);
-          totalTime = hours + "h " + leadingZero(minutes) + "min";
+          if (hours == 0) {
```

Comment:
autant faire une égalité stricte : 
```suggestion
          if (hours === 0) {
```